frontend/simple/js/state.js

```diff
@@ -57,6 +64,24 @@ const mutations = {
       mutations.addContract(state, contract)
     }
   },
+  deleteMail (state, hash) {
+    let mailbox = getMailbox(state)
+    if (mailbox) {
+      let index = state[ mailbox ].messages.findIndex(msg => msg.hash === hash)
+      if (index > -1) {
+        state[ mailbox ].messages.splice(index, 1)
+      }
+    }
+  },
+  readMail (state, hash) {
+    let mailbox = getMailbox(state)
+    if (mailbox) {
+      let index = state[ mailbox ].messages.findIndex(msg => msg.hash === hash)
+      if (index > -1) {
+        state[ mailbox ].messages[index].read = true
+      }
+    }
```

Comment:
If we do above here, we reduce the size of the file by 8 lines just by simplifying these two functions.